api.js

```diff
@@ -715,6 +751,55 @@
     }
   };
 
+  var cachedRoutes = {
+    'get': {
+      '/user/find/(a-zA-Z0-9)': function(urlParts, params) {
+        // Use everything for the hash
+        return [urlParts, orderObject(params)];
+      }
+    },
+    'get-expires': {
+      '/user/find/(a-zA-Z0-9)': 300000 // 300s / 5m
+    }
+  };
+
+  // Returns an ordered version of it's input
+  function orderObject(source) {
+    var output = {};
+    Object.keys(source).sort().forEach(function(key) {
+      output[key] = source[key];
+    });
+    return output;
+  }
+
+  function qhash(input) {
+    switch(typeof input) {
+      case 'string':
+        // do nothing
+        break;
+      case 'object':
+        input = orderObject(input);
+      default:
+        input = JSON.stringify(input);
+        break;
+    }
+
+    if(crypto) {
```

Comment:
You probably recognize it from the server-side cache. Could be removed, because it should never be defined in the browser on this location.